spec/services/hearings_creator_spec.rb

```diff
@@ -328,27 +323,6 @@
     end
   end
 
-  context "when hearing had both a prosecution case and an applicationt to court" do
-    context "with one defendant for the case and one defendant for the application" do
-      it "calls the Sqs::PublishHearing service twice" do
-        allow(LaaReference).to receive(:find_by).with(defendant_id: "dd22b110-7fbc-3036-a076-e4bb40d0a666", linked: true).and_return(defendant_one)
-
-        expect(Sqs::PublishHearing).to receive(:call).once.with(hash_including(shared_time: "2018-10-25 11:30:00",
-                                                                               jurisdiction_type: "MAGISTRATES",
-                                                                               case_urn: "12345",
-                                                                               defendant: defendant_one,
-                                                                               application_data: application,
-                                                                               court_centre_id: "dd22b110-7fbc-3036-a076-e4bb40d0a519"))
-        expect(Sqs::PublishHearing).to receive(:call).once.with(hash_including(shared_time: "2018-10-25 11:30:00",
-                                                                               jurisdiction_type: "MAGISTRATES",
-                                                                               case_urn: "12345",
-                                                                               defendant: defendant_one,
-                                                                               court_centre_id: "dd22b110-7fbc-3036-a076-e4bb40d0a519"))
-        create_hearings
-      end
-    end
-  end
```

Comment:
Do you not think this is worth checking? That it handles a hearing with one of each correctly?